File: go-mux/models/country.go
package models

import (
	"fmt"
)

type Country struct {
	Id    uint    `json:"id"`
	Name  string  `json:"name"`
	Users []*User `gorm:"many2many:user_country_mapping;" json:"-"`
}

func GetCountries() ([]*Country) {
	countries := make([]*Country, 0)
	err := GetDB().Find(&countries).Error
	if err != nil {
		fmt.Println(err)
		return nil
	}

	return countries
}

File: node-express/app/Models/Country.js
const bookshelf = require('../../bookshelf');
const Model = bookshelf.Model;

class Country extends Model
{
    get tableName() {
		return 'countries';
	}

	users() {
		return this.belongsToMany('User', 'user_country_mapping', 'country_id', 'user_id');
	}
}

module.exports = bookshelf.model('Country', Country);
